content/en/docs/tasks/configure-pod-container/security-context.md

```diff
@@ -398,7 +398,7 @@ securityContext:
 To assign SELinux labels to a Container, include the `seLinuxOptions` field in
 the `securityContext` section of your Pod or Container manifest. The
 `seLinuxOptions` field is an
-[SELinuxOptions](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#selinuxoptions-v1-core)
+{{< api-reference page="workloads-resources/container" anchor="SELinuxOptions" >}}
```

Comment:
The `api-reference` shortcode looks more difficult to use. How do you know what string to pass as the anchor string?
Could the anchor string equal, "selinuxoptions"?
Question: Do all resource strings contain a version?